common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1180,17 +1191,77 @@ object RssConf extends Logging {
     }
   }
 
-  def HDDFlusherThread(conf: RssConf): Int = {
-    conf.getInt("rss.flusher.hdd.thread.count", 1)
-  }
+  val WORKER_FLUSHER_HDD_THREAD_COUNT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.flusher.hdd.thread.count")
```

Comment:
As previously discussed, we should unify "thread.count", "numThreads" and so on to "threads"

Reply:
OK， I'll change it.